tcases-ant/pom.xml

```diff
@@ -0,0 +1,112 @@
+<project 
+    xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.cornutum.tcases</groupId>
+    <artifactId>tcases</artifactId>
+    <version>1.5.5-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>tcases-ant</artifactId>
+  <packaging>jar</packaging>
+
+  <name>Tcases Ant</name>
+  <description>Ant support for Tcases</description>
+  <url>https://github.com/Cornutum/tcases</url>
+
+  <developers>
+    <developer>
+      <name>Kerry Kimbrough</name>
+      <email>tcases@cornutum.org</email>
+      <organization>The Cornutum Project</organization>
+      <organizationUrl>http://www.cornutum.org</organizationUrl>
+    </developer>
+  </developers>
+
+  <licenses>
+    <license>
+      <name>The MIT License</name>
+      <url>http://opensource.org/licenses/MIT</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <organization>
+    <name>The Cornutum Project</name>
+    <url>http://www.cornutum.org/</url>
+  </organization>
+
+  <scm>
+    <connection>scm:git:https://github.com/Cornutum/tcases</connection>
+    <developerConnection>scm:git:https://github.com/Cornutum/tcases</developerConnection>
+    <url>https://github.com/Cornutum/tcases</url>
+  </scm>
+
+  <issueManagement>
+    <system>GitHub</system>
+    <url>https://github.com/Cornutum/tcases/issues</url>
+  </issueManagement>
+
+  <mailingLists>
+    <mailingList>
+      <name>Tcases Forum</name>
+      <subscribe>https://groups.google.com/d/forum/tcases</subscribe>
+      <unsubscribe>https://groups.google.com/d/forum/tcases</unsubscribe>
+      <post>https://groups.google.com/d/forum/tcases</post>
+    </mailingList>
+  </mailingLists>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.build.date>${maven.build.timestamp}</project.build.date>
+    <maven.build.timestamp.format>yyyy-MM-dd</maven.build.timestamp.format>
+  </properties>
+
+  <build>
```

Comment:
I copy/pasted the pom.xml files for the new modules. So the lines below are not currently useful, but at least they are consistent.